src/Button.jsx

```diff
@@ -2,13 +2,18 @@ import React from "react";
 import styled from "styled-components";
 
 export const StyledButton = styled.button`
-  text-align: center;
-  background: #322d38;
-  color: white;
-  border-color: #322d38;
-  font-size: 16px;
-  text-decoration: underline;
-  padding: 0;
+  ${props =>
```

Comment:
This is not really the best way to use styled components, all this is saying is if it has the prop `navLink` style this button if else, just use the standard button, are you saying you don't want styling at all?

Reply:
Currently yes, the delete button didn't actually have any proper styling so as that is the only other case where we use the `Button` at the moment we left that blank with the idea that we could add in more conditionals like the one `navLink` uses as and when we need them.

Reply:
I don't think this is best practice but i think you'll have time at the end for stylist changes so I can talk you through maybe what to do then :)